CHANGELOG.txt

```diff
@@ -143,6 +147,8 @@
                     - The constants 'NARROW', 'FLOATING_TIME' & 'FLOATING' are now deprecated;
                       use 'Activity.FloatingType.*' instead.
                     - The constants 'SETTING_*' are now deprecated; use 'activities.setting_keys' instead.
+                    - New FullCalendarBrick base class for implementation of calendar bricks.
+                    - New MyActivitiesCalendarBrick for home view.
```

Comment:
- It's the deprecation section.
- The item 'MyActivitiesCalendarBrick' seems useless in dev section (we've introduced It in user side section).